packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -9,7 +9,7 @@ import { styles } from './RichTextEditor.styles';
 import { FieldExtensionSDK, FieldConnector } from '@contentful/field-editor-shared';
 import schema from './constants/Schema';
 import deepEquals from 'fast-deep-equal';
-import debounce from 'lodash/debounce';
+// import debounce from 'lodash/debounce';
```

Comment:
Remove it temporarily until we figure out whether we need it or not because this was causing some actions to act slow on some user cases.